exporting/pubsub/metadata.yaml

```diff
@@ -0,0 +1,156 @@
+# yamllint disable rule:line-length
+---
+id: 'export-google-pubsub'
+meta:
+  name: 'Google Cloud Pub Sub'
+  link: 'https://cloud.google.com/pubsub'
+  categories:
+    - export
+  icon_filename: 'googlecloud.png'
+keywords:
+  - exporter
+  - Google Cloud
+  - Pub Sub
+overview:
+  exporter_description: |
+    Export metrics to Google Cloud Pub/Sub Service
+  exporter_limitations: ''
+setup:
+  prerequisites:
+    list:
+      - title: ''
+        description: |
+          - First [install](https://github.com/googleapis/google-cloud-cpp/) install Google Cloud Platform C++ Client Libraries
+          - Pub/Sub support is also dependent on the dependencies of those libraries, like `protobuf`, `protoc`, and `grpc`
+          - Next, Netdata should be re-installed from the source. The installer will detect that the required libraries are now available.
+  configuration:
+    file:
+      name: 'exporting.conf'
+    options:
+      description: |
+        The following options can be defined for this exporter.
+      folding:
+        title: 'Config options'
+        enabled: true
+      list:
+        - name: 'enabled'
+          default_value: 'no'
+          description: 'Enables or disables an exporting connector instance (yes|no).'
+          required: true
+        - name: 'destination'
+          default_value: 'pubsub.googleapis.com'
+          description: 'Accepts a space separated list of hostnames, IPs (IPv4 and IPv6) and ports to connect to. Netdata will use the first available to send the metrics.'
+          required: true
+          detailed_description: |
+            The format of each item in this list, is: [PROTOCOL:]IP[:PORT].
+            - PROTOCOL can be udp or tcp. tcp is the default and only supported by the current exporting engine.
+            - IP can be XX.XX.XX.XX (IPv4), or [XX:XX...XX:XX] (IPv6). For IPv6 you can to enclose the IP in [] to separate it from the port.
+            - PORT can be a number of a service name. If omitted, the default port for the exporting connector will be used.
+
+            Example IPv4:
```

Comment:
We are not using default value as example and we specify a complete different port. I suggest only use `default value` , instead to use the same example used in others.

Reply:
@thiagoftsm: This is a Config options section and I guess it is okay to give some common example. The specific example will appear in the Config Examples section below.